### DISCUSSION TITLE
Parallel execution of theory arguments

### DISCUSSION THREAD

**Discussion by LukasKarel** [xunit/xunit#2235] [Question] [answered]
I'm using xunit to test communication between different applications in our system. Because of sequential dependencies I cant execute all tests of one collection in parallel. But a theory test could be executed in parallel with all its arguments. Is there a possibility to perform theory tests in parallel?

**Comment by bradwilson** [comment, marked as answer]
There's no way to do this today in v2, and given your description, I'm not convinced that we will have what you're looking in v3, either. You have an odd hybrid of parallelism/non-parallelism going on here that is I think probably unique to your problem area (or at least to the way you've chosen to test it).
